ts/query_test.go

```diff
@@ -224,9 +266,16 @@ func (tm *testModel) assertQuery(name string, r Resolution, start, end int64,
 	}
 	iters.init()
 	for iters.isValid() {
+		var value float64
+		switch q.GetAggregator() {
+		case proto.TimeSeriesQueryAggregator_AVG:
+			value = iters.avg()
+		case proto.TimeSeriesQueryAggregator_AVG_RATE:
+			value = iters.dAvg()
+		}
```

Comment:
Sorry, this was implemented in both the actual code and the test model, this comment was on the model implementation but I thought pete was commenting on the production implementation.  The production implementation was updated per pete's suggestion.